Uplift/Views/SportsDetail/SportsDetailHeaderView.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  SportsDetailHeaderView.swift
+//  Uplift
+//
+//  Created by Artesia Ko on 4/18/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol SportsDetailHeaderViewDelegate: class {
+    func sportsDetailHeaderViewBackButtonTapped()
+}
+
+class SportsDetailHeaderView: UICollectionReusableView {
+    
+    private let backButton = UIButton()
+    private let imageView = UIImageView()
+    private let titleLabel = UILabel()
+    private let locationLabel = UILabel()
+    private let typeLabel = UILabel()
+    private let nameLabel = UILabel()
+    
+    private weak var delegate: SportsDetailHeaderViewDelegate?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupViews()
+        setupConstraints()
+    }
+    
+    func configure(for delegate: SportsDetailHeaderViewDelegate, for post: Post) {
+        self.delegate = delegate
+        
+        titleLabel.text = post.title.uppercased()
+        locationLabel.text = post.location
+        typeLabel.text = post.type.uppercased()
+        nameLabel.text = "Created by: TODO"
```

Comment:
Can we add this to `ClientStrings` too?

Reply:
will be replaced by post creator's name when models are updated.